### PR TITLE
[Helix] Reenable win8 helix queue

### LINKED ISSUE #21421: Restore use of Windows.81.Amd64.Open queue
See removal in #21420. Should be able to restore use in about a week.

### PR DESCRIPTION
**Pull request by HaoK**

### CODE REVIEW

**Comment by HaoK** [comment]
Note: we were running OSX helix queues in both PRs and daily, which is just double running them on rolling builds.  Fixing that in this PR too

File: eng/targets/Helix.Common.props [hunk 1]
```diff
@@ -27,8 +27,8 @@
 
   <!-- daily scheduled only queues -->
   <ItemGroup Condition="'$(IsWindowsOnlyTest)' != 'true' AND '$(TargetArchitecture)' == 'x64' AND '$(IsHelixDaily)' == 'true'">
+    <HelixAvailableTargetQueue Include="Windows.81.Amd64.Open" Platform="Windows" />
     <HelixAvailableTargetQueue Include="Ubuntu.1804.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="OSX.1014.Amd64.Open" Platform="Linux" />
```
Comment (dougbu) [inline eng/targets/Helix.Common.props:31]: Just curious: Do we have a `macOS` or `OSX` platform? Seems less like our Linux platforms every day 😈

**Comment by dougbu** [comment]
@HaoK are the failures due to flaky tests and not a Win8 specific problem? BTW should close dotnet/aspnetcore#21421 when this works 😆

**Comment by HaoK** [comment]
That's a good question, I don't even see any test failures in the openapi test logs, just 26 tests passed and 4 skipped :/ https://helix.dot.net/api/2019-06-17/jobs/67e5400f-917e-4779-8e23-ae8f52d8053f/workitems/dotnet-microsoft.openapi.Tests--netcoreapp5.0/files/console.0ff64487.log

File: eng/targets/Helix.Common.props (hunk 1 above)
Comment (HaoK) [reply eng/targets/Helix.Common.props:31]: Platform is really windows or not windows at this point :)

**Comment by HaoK** [comment]
Win8 has looked fine, reverting yaml changes

**Review by Pilchie — APPROVED** [review]

**Pull request merged**